### ISSUE TITLE
Data URI error in max-len

### ISSUE THREAD

**Issue by allan2coder** [vuejs/eslint-plugin-vue#1005]
**Tell us about your environment**	
* **ESLint version:**	6.1.0
* **eslint-plugin-vue version:**	5.2.2
* **Node version:** 	10.16.0

**Please show your full configuration:**

```json
default
```

**What did you do?**	

```vue
<template>...</template>
<script>...</script>
<style lang="less">
.test{
    background: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAMAAABg3Am1AAAAh1BMVEUAAAD/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWX/SWV+ZsJgAAAALHRSTlMA/PPx7xXqyLRrD+VOh28lCgP1zT4rGgbAjHdcOicf0mXc2LqtkIFhVzagSSpfOkgAAAG7SURBVEjHlZbXloMgEEAHxRKj0TW9F7Op8//ftycFnAHWct843jkhMAWwmN2W8Sksy/AUL28BtJAMImREg6aYIPPQwlvOwM3PyEcn3mru8ndj/JdxbvtPiQ14qelfBTYiDtxfYysD6qcCWxEp+b8SO+Bt9XlG2Im9Ot0BdmT1vV96X7FxXENJN/W58wxrFgVcmZ/fS7L8feebT3aZqB0qn59gGQDAoV4fPz850v7WvKMRACzq5QaARoRbtdSMARJBdqRyf/X2p9/VhCg7SFlWJl/nQn36t9eQ8QLTEbXvITunM/KIQpW208cK9qjQN0HZGFV4hCEaVEWDjxLsOo7rCr5bX4UjAB/Kz+2Pwt4S+ned9w80kbB3+TrijAZHOJv+BqAhooLM8hnzGJFfXMrrVvnTiztiDYmg/kT5IWbwoaiQsGPpXRIf8aIiiDHmBZRqn9Y8THgBJb6ZR9uQfAbIQ16i9Jzin5c/ZO0xOBpNAALJsiKnd3+AIqJtxm5kGfNRXOmprlytUjaMlXnvZqx5dmr3/QcKi2gdWf2Gop86xm7UNnbtwS7bB3v/p0P74ySBFoLX80cKId3Pnz84CvgkMbBytAAAAABJRU5ErkJggg==";
}
</style>
```

**What did you expect to happen?**
No error for Data URI

**What actually happened?**

`error This line has a length of 989. Maximum allowed is 120 max-len`

**Comment by ota-meshi** [comment]
Thank you for this issue.

I add the `vue/max-len` rule. #959
This rule only validates inside `<template>` and `<script>`.
I will release this rule soon.

`vue/max-len` rule doesn't validate the line length of `<style>`, but If you want to validate the line length of `<style>`, I think you can use stylelint.

https://stylelint.io/user-guide/rules/max-line-length

**Comment by ota-meshi** [comment]
I have released v6.1.0. Please use v6.1.0 and the `vue/max-len` rule.
